### PR TITLE
Add notes for betas for Chocolatey components

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add release notes for beta releases of all Chocolatey components

## Motivation and Context

There are releases coming up, and we need to get this documented

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

**Review by JPRuskin — COMMENTED** [review]
Good catch on the last round of CLI bits.
One minor note on one of the issue titles, I think?

File: input/en-us/choco/release-notes.md [hunk 1]
```diff
@@ -19,6 +19,34 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+> If you run into any problems when using this beta version of Chocolatey CLI we would ask that you comment on this [discussion](https://github.com/chocolatey/choco/discussions/2995), which is where we will be collating issues, and providing workarounds, etc.  We will not be accepting issues raised against this beta release.
+
+### Known Issues
+
+See this [list](https://github.com/chocolatey/choco/discussions/2995) for known issues with this pre-release.
+
+### Bugs
+
+- Chocolatey does not override NuGet proxy with its own - see [#1720](https://github.com/chocolatey/choco/issues/1720).
+- When no hook scripts and installed you cannot install/upgrade a package on a system using PowerShell 2 - see [#3088](https://github.com/chocolatey/choco/issues/3088).
```
[the remark this answers is not in the document]
Comment (gep13) [reply input/en-us/choco/release-notes.md]: Doh! Thanks for catching that! I have fixed that up just now.

**Review by AdmiringWorm — DISMISSED** [review]
LGTM

**Review by JPRuskin — APPROVED** [review]

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Pull request merged**